docs/io/fits/usage/headers.rst

```diff
@@ -47,7 +47,7 @@ Value Access, Updating, and Creating
 """"""""""""""""""""""""""""""""""""
 
 As shown in the :ref:`Getting Started <tutorial>` tutorial, keyword values can
-be accessed via keyword name or index of an HDU's header attribute. Here is a
+be accessed via keyword name or index of an HDU's header attribute. You can also use the wildcard character ``*`` to get the keyword value pairs that match your search string. Here is a
```

Comment:
Even if RST does not get PEP8 check, I would recommend the line not to be too long. I like the 79 char limit but some others might prefer 100.